internal/alb/lb/loadbalancer.go

```diff
@@ -41,26 +39,22 @@ func NewController(
 	store store.Storer,
 	nameTagGen NameTagGenerator,
 	tgGroupController tg.GroupController,
-	lsGroupController ls.GroupController,
-	sgAssociationController sg.AssociationController,
 	tagsController tags.Controller) Controller {
 	attrsController := NewAttributesController(cloud)
 	wafController := NewWAFController(cloud)
```

Comment:
Can we get rid of the waf and shield controllers too since it looks like their reconcilers have also been commented out?